packages/python/README.md

```diff
@@ -69,11 +74,16 @@ First you'll need to write a "grouping function" to inform ReadMe of the user or
 ```python
 def grouping_function(request):
     # You can lookup your user here, pull it off the request object, etc.
-    return {
-        "api_key": "unique api_key of the user",
-        "label": "label for us to show for this user (ie email, project name, user name, etc)",
-        "email": "email address for user"
-    }
+    # Your grouping function should None if you don't want the request to be
+    # logged, and otherwise return a structure like the one below.
```

Comment:
```suggestion
    # Your grouping function should return None if you don't want the request
    # to be logged, and otherwise return a structure like the one below.
```